Makefile

```diff
@@ -17,6 +17,15 @@ integration-tests:
 	fi
 	@./integration-tmp/bin/bats ./test/integration/dnet
 
+./cmd/dnet/dnet:
+	make build
```

Comment:
Not sure how it escaped me before. But here you probably don't need to spawn the build inside the container. You can change this to `make build-local`